.github/workflows/lint.yml

```diff
@@ -48,6 +48,7 @@ jobs:
           reporter: github-pr-check
           level: warning
           flags: --linelength=120
+          filter: "-runtime/references"
```

Comment:
Is this for this specific error-message? Just wondering, since the name sounds too generic. Thanks!

Reply:
I think so: https://github.com/cpplint/cpplint/issues/106